bindings/sphinxdoc/docs/source/boolean_ops.rst

```diff
@@ -0,0 +1,7 @@
+boolean\_ops module
```

Comment:
This should not be here *(part of other PR)*. This applies to all files in this folder.

Reply:
Ok, that got messed up, I believe the whole folder should not be there?

Reply:
Yes